src/data/admin/Mentors.tsx

```diff
@@ -19,7 +65,7 @@ export const TAGS = [
   },
 ];
 
-export const COLUMNS = [
+export const COLUMNS: column[] = [
```

Comment:
check tanstack table for the column defs: ColumnDef

Reply:
should i be changing the cell property or all the properties? Should i be importing tanstack?